stock/templates/registered/fleetupfitting.html

```diff
@@ -0,0 +1,152 @@
+                        </div>
+                        <div class="col-lg-8 col-lg-offset-7">
+                            <a class="btn btn-primary" href="/fleetup/doctrines/{{ doctrin.DoctrineId }}/">See doctrine</a>
+                        </div>
+                        {% endfor %}
+                    </div>
+                </div>
+                <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h3 class="panel-title">Fit categories</h3>
+                        </div>
+                        <div class="panel-body">
+                            {% for category in y.Categories %}
+                            <span class="label label-success">{{ category }}</span>
+                            {% endfor %}
+                        </div>
+                </div>
+                {% endfor %}
+                <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h3 class="panel-title">All fits in this Doctrine</h3>
+                        </div>
+                        <div class="panel-body">
+                            <div class="list-group">
+                            {% for arbit, orbit in doctrines_list.items %}
+                              
+                                {% for fitting in orbit.Data %}
+                                <a href="/fleetup/fittings/{{ fitting.FittingId }}/" class="list-group-item">
+                                
+                                <h4 class="list-group-item-heading">{{ fitting.Name }}<span class="pull-right"><img src="https://image.eveonline.com/InventoryType/{{ fitting.EveTypeId }}_32.png" class="img-circle"></span></h4>
+                                <p class="list-group-item-heading">{{ fitting.Role }} - {{ fitting.ShipType }}</p>
+                                </a>
+                            {% endfor %}
+                            {% endfor %}
+                            </div>
+                        </div>
+                </div>
+            </div>
+            <div class="col-lg-4">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        {% for a, j in fitting_data.items %}
+                        <h3 class="panel-title">{{ j.Name }}</h3>
+                    </div>
+                    <div class="panel-body">
+                        <div class="col-lg-3">
+                            <img src="https://image.eveonline.com/InventoryType/{{ j.EveTypeId }}_64.png" class="img-responsive">
+                        </div>
+                        <div class="col-lg-9">
+                            <p>Hull: <b>{{ j.HullType }}</b></p>
+                            <p>Ship: <b>{{ j.ShipType }}</b></p>
+                            {% load humanize %}
+                            <p>Estimated price: <b>{{ j.EstPrice|intword }} ISK</b></p>
```

Comment:
wouldn't intcomma be more accurate? for instance the SRP uses intcomma for the losses.